packages/tupaia-web-server/src/routes/EntityRoute.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import { Route } from '@tupaia/server-boilerplate';
+import camelCaseKeys from 'camelcase-keys';
+
+// TODO: WAITP-1278 Move this to types
+export type EntityRequest = Request<any, any, any, any>;
```

Comment:
#4731 is in progress but having some issues in codeship to fixup, if we merge this first, I'll make the update on that branch